api/src/test/java/uk/gov/dhsc/htbhf/claimant/testsupport/VoucherEntitlementTestDataFactory.java

```diff
@@ -16,6 +16,16 @@ public static VoucherEntitlement aValidVoucherEntitlement() {
                 .build();
     }
 
+    public static VoucherEntitlement aVoucherEntitlementWithZeroVouchers() {
+        return VoucherEntitlement.builder()
+                .voucherValueInPence(VOUCHER_VALUE_IN_PENCE)
```

Comment:
I keep mistaking this field for the total voucher value (I thought that its value should have been zero and took me a while to figure out why it wasn't the case). Might it be worthwhile to rename this to something like `individualVouchValueInPence`? Obviously not for this PR though...